update.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -e
+
+msg () {
+	1>&2 echo $*
+}
+
+# check that we are in a prototype
+if ! grep -q govuk-prototype-kit package.json 2> /dev/null; then
```

Comment:
per offline discussion, it's _probably_ better to actually parse the JSON, using `JQ` or i.e. shelling out to `node` as this can give us false positives, and JSON parsing with `grep` is something I would really try to avoid

Reply:
I see your point, I don't want to require the user to install `jq` though.

Also, as this is a safety check, I'm more worried about false negatives than false positives, i.e. if the check is accidentally preventing the script from running that's better than the script running when it shouldn't. In testing I haven't seen any false negatives (or false positives), so am tempted to leave this as is.

Reply:
maybe this is slightly safer to check for `"govuk-prototype-kit"` - then it wouldn't catch any other similarly named packages